vic/drivers/shared_all/src/compute_derived_state_vars.c

```diff
@@ -46,13 +46,20 @@ compute_derived_state_vars(all_vars_struct *all_vars,
     size_t                     veg;
     size_t                     lidx;
     size_t                     band;
+    size_t                     tmpTshape[] = {
+        MAX_LAYERS, MAX_NODES,
+        MAX_FROST_AREAS + 1
+    };
+    size_t                     tmpZshape[] = {
+        MAX_LAYERS, MAX_NODES
+    };
```

Comment:
Indeed - I think it's ugly - but uncrustify is going to change this every time, so I'll just leave it.